assets/src/stories-editor/blocks/amp-story-page-attachment/attachment-opener.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { RichText } from '@wordpress/block-editor';
+import { ENTER, SPACE } from '@wordpress/keycodes';
+
+const AttachmentOpener = ( { setAttributes, toggleAttachment, openText } ) => {
+	return (
+		<div className="open-attachment-wrapper">
+			<span
+				role="button"
+				tabIndex="0"
+				onClick={ () => {
+					toggleAttachment( true );
+				} }
+				onKeyDown={ ( event ) => {
+					if ( ENTER === event.keyCode || SPACE === event.keyCode ) {
+						toggleAttachment( true );
+						event.stopPropagation();
+					}
+				} }
+				className="amp-story-page-open-attachment-icon"
+			>
+				<span className="amp-story-page-open-attachment-bar amp-story-page-open-attachment-bar-left" />
+				<span className="amp-story-page-open-attachment-bar amp-story-page-open-attachment-bar-right" />
+			</span>
+			<RichText
```

Comment:
Should this attachment text have option to style the font?

Reply:
I don't think it makes sense to have a font selection option for this kind of embedded content. It wasn't mentioned in #1973 nor #2782, so I consider it out of scope for this PR.

Reply:
I believe that publishers are going to want to style all of text on a story with they font. It would extremely strange if some part do not have the brand styling. I have created a ticket regarding this. https://github.com/ampproject/amp-wp/issues/3224